src/dayview.css

```diff
@@ -113,13 +135,21 @@
   margin-right: 50px;
 }
 
-.dayview-allday-content-table {
-  min-height: 50px;
+.dayview-allday-category-wrapper {
```

Comment:
not sure you still need this, as we already have style defined as below

.dayview-allday-content-wrapper {
  margin-left: 50px;
  overflow: hidden;
  height: 51px;
}

Reply:
it's used to override height part of dayview-allday-content-wrapper. The new wrapper shifts the height 1px up so that the borders of allday content table can overlap with existing borders of above category table and below content table.
Without the override, the top and bottom borders look like duplicated:
![image](https://github.com/twinssbc/Ionic2-Calendar/assets/27603495/e28cb121-627f-4956-957f-e7f6c368800f)